client/src/modules/reports/generate/stock_exit/stock_exit.html

```diff
@@ -0,0 +1,90 @@
+<bh-report-preview
+  ng-if="ReportConfigCtrl.previewGenerated"
+  source-document="ReportConfigCtrl.previewResult"
+  on-clear-callback="ReportConfigCtrl.clearPreview()"
+  on-save-callback="ReportConfigCtrl.requestSaveAs()">
+</bh-report-preview>
+
+<div ng-show="!ReportConfigCtrl.previewGenerated">
+  <div class="row">
+    <div class="col-md-12">
+      <h3 translate>REPORT.STOCK.EXIT_REPORT</h3>
+      <p class="text-info" translate>REPORT.STOCK.EXIT_REPORT_DESCRIPTION</p>
+    </div>
+  </div>
+
+  <div class="row" style="margin-top : 10px">
+    <div class="col-md-6">
+      <div class="panel panel-default">
+        <div class="panel-heading">
+          <span translate>REPORT.UTIL.OPTIONS</span>
+        </div>
+
+        <div class="panel-body">
+
+        <form name="ConfigForm" bh-submit="ReportConfigCtrl.preview(ConfigForm)" novalidate autocomplete="off">
+
+          <!-- select depot -->
+          <bh-depot-select
+            depot-uuid="ReportConfigCtrl.depot.uuid"
+            on-select-callback="ReportConfigCtrl.onSelectDepot(depot)"
+            required="true"
+            validate-trigger="ConfigForm.$submitted">
+            <bh-clear on-clear="ReportConfigCtrl.clear('depot')"></bh-clear>
+          </bh-depot-select>
+
+          <!-- date interval -->
+          <bh-date-interval 
+            date-from="ReportConfigCtrl.dateFrom" 
+            date-to="ReportConfigCtrl.dateTo">
+          </bh-date-interval>
+
+          <!-- stock exit type -->
+          <div class="checkbox">
+            <label>
+              <input type="checkbox" ng-true-value="1" ng-false-value="0" value="1" ng-model="ReportConfigCtrl.includePatientExit">
```

Comment:
Did you mean to have both `ng-*-value` and `value="1"`?  If you want to make this "checked" by default, I recommend you just set the model to `1` and let AngularJS do the work for you :)